bundle/src/main/java/com/adobe/acs/commons/properties/util/TemplateReplacementUtil.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2019 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.properties.util;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Util class used to provide helper methods for finding and replacing the tokens used in this feature.
+ */
+public class TemplateReplacementUtil {
+
+    private TemplateReplacementUtil() {
+    }
+
+    private static final String PLACEHOLDER_BEGIN = "{{";
```

Comment:
nit: by convention I believe vars (both static and non) should be above ctor declaration

Reply:
addressed in the regex refactor